weaver/wps_restapi/swagger_definitions.py

```diff
@@ -228,6 +233,23 @@ class URL(ExtendedSchemaNode):
     format = "url"
 
 
+class DateTimeInterval(ExtendedSchemaNode):
+    schema_type = String
+    description = "DateTime format against OGC-API - Processes,\n\
+        to get values before a certain date-time use '../' before the date-time,\n\
+        to get values after a certain date-time use '/..' after the date-time like the example,\n\
+        to get values between two date-times use '/' between the date-times,\n\
+        to get values with a specific date-time just pass the datetime"
```

Comment:
Result format still doesn't appear to be good in the resulting JSON schema. 
It has a lot of useless `     \n`. 

Add `.` at the end.